packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Client/Naming/NamingTests.cs

```diff
@@ -1,109 +1,111 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System.Threading.Tasks;
-using Client.Naming;
-using Client.Naming.Models;
-using NUnit.Framework;
-using ClientModel = Client.Naming.Models.ClientModel;
+// using System.Threading.Tasks;
```

Comment:
these test cases are removed because of this issue: https://github.com/microsoft/typespec/issues/5653

Reply:
Can we link to the GH issue in this file?

Reply:
done